class_defines.py

```diff
@@ -2402,10 +2400,12 @@ def init_props(self, **kwargs):
         self.value = value
         return value, None
 
+    def text_inside(self, ui_scale):
+        return (ui_scale * abs(self.draw_outset)) < self.value/2
+
```

Comment:
I cannot say _why_ this requires ui_scale to track properly.  Tested under windows 11 with display scaling property @ 175%,

Reply:
update: the inconsistency comes from chasing a pre-existing inconsistency.   I'll see if I can expunge the original inconsistency without breaking anything else.  I see what I have to do but it'll take some time to test.  I propose allowing this ugly odd argument to be ---for now.